contrib/core/actions/pause.yaml

```diff
@@ -0,0 +1,18 @@
+---
+  name: "pause"
+  runner_type: "run-python"
+  description: "Action to Pause workflow"
```

Comment:
Yeah, it will pause "everything" - well just the actual action execution will behave as sleep, but it has little use outside of workflows... :)